app/src/main/java/com/example/mona/facebookoffline/EditActivity.java

```diff
@@ -19,18 +20,32 @@
     @Inject FacebookApiService mFacebookApiService;
 
     private EditText mPostBody;
+    private EditText mPostTitle;
+    private Post post;
```

Comment:
nit: by convention, member variables should be prefixed with an `m`. Like `mPost`